src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -461,6 +461,10 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 			this._model.updateActiveCell(undefined);
 		}
 	}
+
+	private isWYSIWYGMode(): boolean {
+		return this.cellModel && this.cellModel.showPreview && !this.cellModel.showMarkdown && this.cellModel.isEditMode;
```

Comment:
Seems like something that should probably go into cellModel - having something like `currentMode` would be pretty useful I think and being explicit about mode changes makes sense to me instead of relying on the combination of a bunch of different states.